src/software/simulation/physics/physics_world.h

```diff
@@ -18,13 +18,19 @@
 class PhysicsWorld
 {
    public:
+    // TODO: where shoudl this go? the world/ folder? here?
+    typedef struct RobotStateWithId_t
+    {
+        unsigned int id;
+        RobotState robot_state;
+    } RobotStateWithId;
```

Comment:
Thoughts on RobotState containing ID?

Reply:
I guess that would be fine. So far my physics use cases need an ID. @akhilveeraghanta @EvanMorcom @garethellis0

Reply:
I will move to robot_state.h for now and see how things go

Reply:
I wil also reverse the _t stuff

Reply:
done